rpc/common/moc_common_common.proto

```diff
@@ -253,9 +253,10 @@ enum ProviderAccessOperation {
 	VirtualMachine_Start 				= 604;
 	VirtualMachine_Stop 				= 605;
 	VirtualMachine_Reset 				= 606;
-	VirtualMachine_Repair_Guest_Agent		= 607;
+	VirtualMachine_Repair_Guest_Agent  		= 607;
 	VirtualMachine_Pause 				= 608;
 	VirtualMachine_Save 				= 609;
+	VirtualMachine_Remove_Iso_Disk		= 610;
```

Comment:
nit: spacing, not sure if the build will adjust it

Reply:
Fixed